robot/server/server.go

```diff
@@ -445,6 +445,17 @@ func (s *Server) Log(ctx context.Context, req *pb.LogRequest) (*pb.LogResponse,
 	fields := make([]zapcore.Field, 0, len(log.Fields)*2)
 	for _, fieldP := range log.Fields {
 		field, err := logging.FieldFromProto(fieldP)
+
+		// Handle poorly serialized error fields (force them into a string type
```

Comment:
Do we want to put this here, or in `FieldFromProto`? I guess it doesn't matter too much given the limited code paths using `FieldFromProto`.

Reply:
Good point. Probably `FieldFromProto` since the client-side analagous code is in `FieldToProto`.